templates/kubernetes-cluster-with-new-vpc.template

```diff
@@ -441,7 +441,7 @@ Parameters:
     AllowedPattern: ^[0-9a-zA-Z-/]*$
     ConstraintDescription: Quick Start key prefix can include numbers, lowercase
       letters, uppercase letters, hyphens (-), and forward slash (/).
-    Default: quickstart-heptio/
+    Default: quickstart-vmware/
```

Comment:
Do we need to copy assets from an older public s3 bucket to this new one?

Reply:
I don't necessarily think we need to transfer any old assets as long as we deploy the currently needed assets as part of the initial deployment.

Reply:
By the looks of it via the aws cli when AWS did their rename they handled this. The bucket exists and has reasonable data in it already.